Flepper.QueryBuilder/Utils/Extensions/ColumnsExtensions.cs

```diff
@@ -1,16 +1,44 @@
-﻿using System.Linq;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace Flepper.QueryBuilder.Utils.Extensions
 {
     internal static class ColumnsExtensions
     {
+        private const string ALIAS = " AS ";
+
         public static string JoinColumns(this string[] columns)
         {
-            var fields = columns.Aggregate("", (current, column) => current + $"[{column}],");
-
+            var newColumns = GetAllColumnsWithAlias(columns);
+            var fields = newColumns.Aggregate("", (current, column) => current + $"{column},");
             fields = fields.Remove(fields.Length - 1, 1) + " ";
-
             return fields;
         }
+
+        private static List<string> GetAllColumnsWithAlias(string[] columns)
+        {
+            var newColumns = new List<string>(columns.Length);
+            foreach (var column in columns)
+            {
+                if (ContainsIgnoreCase(column, ALIAS))
```

Comment:
When I read **ContainsIgnoreCase** I think that it will check if something contains a "IgnoreCase", but when I saw the implementation, I understod that, so **ContainsIgnoringCase** would be a better name.